packages/react/src/layouts/side-nav-outer-toolbar/side-nav-outer-toolbar.tsx

```diff
@@ -35,20 +38,20 @@ export const SideNavOuterToolbar = ({ title, children }: React.PropsWithChildren
   }, [isLarge]);
 
   const onNavigationChanged = useCallback(
-    ({ itemData: { path }, event, node }) => {
-      if (menuStatus === MenuStatus.Closed || !path || node.selected) {
-        event.preventDefault();
+    ({ itemData: { path }, event, node }: TreeViewTypes.ItemClickEvent & { itemData: SideNavigationItem }) => {
```

Comment:
Can't we use generic here for itemData?

Reply:
No, no generic parameter `TItem`.
Compare: 
List
<img width="896" alt="image" src="https://github.com/DevExpress/devextreme-ui-template-gallery/assets/11459924/486dc7db-f790-42a8-ad3e-1b60c3c49826">

Tree View:
<img width="893" alt="image" src="https://github.com/DevExpress/devextreme-ui-template-gallery/assets/11459924/1a115fcd-f7fd-4c48-8676-3aa5b99ffcbb">